pkg/autoops/api/api.go

```diff
@@ -706,7 +706,7 @@ func (s *AutoOpsService) UpdateAutoOpsRule(
 			return err
 		}
 
-		if autoOpsRule.AlreadyTriggered() || autoOpsRule.IsFinished() || autoOpsRule.IsStopped() {
+		if autoOpsRule.IsFinished() || autoOpsRule.IsStopped() {
```

Comment:
How about adding a new method `IsCompleted()` to the domain?

Reply:
The `isCompleted` will be confusing IMHO.
Because the complete and finish have the same meaning in this case.
It will be easier to understand and leave it as it is. WDYT?

Reply:
@cre8ivejp @kentakozuka 

It may be confusing how to use it.
Add deleted to the condition,
How about changing it to `IsInactive()`?

Reply:
@kakcy, the issue is not the function name. By creating `IsInactive` or `isCompleted`, we are creating a new state that doesn't exist in the auto_ops_rule domain layer, which is confusing because  `completed` and `finished` words have the same meaning in this case.
So, my suggestion is to leave it as it is.

Reply:
@cre8ivejp @kentakozuka 
Since there were multiple locations that made the same judgment, we decided that it would be better to add a new one to improve the readability of the code.

However, as you pointed out, `IsCompleted()` does not exist in AutoOpsStatus, and having a function that determines a similar status causes confusion when used, so I will discontinue it.